test/Intl/DateTimeFormat.js

```diff
@@ -323,14 +323,17 @@ const tests = [
             // TODO(jahorto): re-enable the commented out GMT/UTC tests once we have a way
             // to assert different expected values depending on CLDR version
             // ICU ~55 formats GMT-like time zones as GMT, but ICU ~60 formats them as UTC
-            // test(newYears, "GMT", "short", "UTC", "UTC");
-            // test(newYears, "GMT", "long", "Coordinated Universal Time", "UTC");
-            // test(newYears, "Etc/GMT", "short", "UTC", "UTC");
-            // test(newYears, "Etc/GMT", "long", "Coordinated Universal Time", "UTC");
-            // test(newYears, "Etc/UTC", "short", "UTC", "UTC");
-            // test(newYears, "Etc/UTC", "long", "Coordinated Universal Time", "UTC");
-            // test(newYears, "Etc/UCT", "short", "UTC", "UTC");
-            // test(newYears, "Etc/UCT", "long", "Coordinated Universal Time", "UTC");
+            if (WScript.Platform.INTL_LIBRARY_VERSION >= 60) {
```

Comment:
Isn't this named `ICU_VERSION`?

Reply:
Right yeah, I renamed it midway through. Good catch.